Hymns/Display/DisplayHymnView.swift

```diff
@@ -16,6 +19,11 @@ struct DisplayHymnView: View {
             if !viewModel.isLoaded {
                 ActivityIndicator().maxSize()
             } else {
+                ZStack {
+                    if self.initiatedSoundCloud {
```

Comment:
Can you have consistent naming of variables? Pick either soundCloudinitiated or initiatedSoundCloud (although frankly both of them are kind of bad lol. "initiated" isn't really an accurate description of what it's doing. Maybe `isSoundCloudBackgrounded` is more informative

Reply:
Thanks! great feedback. I went ahead and made the change here in recent commit